varnish.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	vbeReload       = "VBE.reload_"
+	vbeReloadLength = len(vbeReload)
```

Comment:
You can do this inline in `findMostRecentVbeReloadPrefix`, no need to make this globals constant.

Reply:
Ok, it was maybe and excessive attempt of optimization on my side. :D
Done: feb61723705840fcea26a8e76be989e666ec8198